server/service/src/programs/schemas/patient.json

```diff
@@ -266,7 +273,10 @@
           "$ref": "#/definitions/SocioEconomics"
         }
       },
-      "required": ["contactDetails", "id"],
+      "required": [
+        "contactDetails",
+        "id"
+      ],
       "type": "object"
     },
```

Comment:
This all seems like whitespace changes, and I don't think it's the standard JSON formatting (the previous was), this seems like the output of the programschemas repo before re-formatting, so probably best to revert formatting?